includes/models/class-payment.php

```diff
@@ -287,7 +287,7 @@ public function get_checkout_url( $force_http = false ) {
         $url = wpbdp_url( 'checkout', $this->payment_key );
 
         if ( ! $force_http && ! is_ssl() ) {
-            $url = set_url_scheme( $url, 'https' );
+            //$url = set_url_scheme( $url, 'https' );
```

Comment:
Why is this commented out?

Reply:
Forgot to comment it back, its for my local tests